libflux/flux/benches/benchmarks.rs

```diff
@@ -1070,6 +1070,16 @@ builtin table : (<-tables: [A], columns: C) => [B] where A: Record, B: Record, C
 //         The length of the interval. This defaults to the
 //         every duration.
 builtin window : (<-tables: [A], ?time: string, every: duration, ?period: duration, columns: C) => [B] where A: Record, B: Record, C: Record
+builtin window2 : (
+    <-tables: [{T with _time: time}]
+    ?every: duration,
+    ?period: duration,
+    ?offset: duration,
+    ?createEmpty: bool
+) => [{A with
+    _start: time, 
+    _stop: time,
+    _time: time}]
```

Comment:
@rockstar Thoughts? Should we be trying to keep these benchmarks files up to date when we make changes? Or should we consider them static now?

Reply:
I guess I must've missed when this got added. That's a very big long string that we don't necessarily need in the source. We should change line 23 to be `const LONGER: &'static str = include_str!("path/to/everything.flux")` like we do in other benchmarks to embed it at compile time. I realize it's not strictly part of this PR, but that might be appropriate to do now, since we have to touch it; it'll prevent others from having to make that decision when it needs to be changed next time.

Reply:
@rockstar That sounds like a good idea. I'll make that change in this PR.